.travis.yml

```diff
@@ -37,54 +37,60 @@ submit_to_coverity_scan: &submit_to_coverity_scan
                "https://scan.coverity.com/builds";
   fi
 
-linux_gcc8: &linux_gcc8
+ubuntu1804_gcc10: &ubuntu1804_gcc10
   os: linux
-  dist: xenial
+  dist: bionic
   compiler: gcc
   addons:
     apt:
       update: true
-      sources: [ ubuntu-toolchain-r-test ]
-      packages: [ gcc-8, g++-8 ]
+      sources:
+        - sourceline: 'ppa:ubuntu-toolchain-r/test'
+      packages: [ gcc-10 g++-10 ]
   before_install:
-    - eval "export CC=gcc-8"
-    - eval "export CXX=g++-8"
+    - eval "export CC=gcc-10 CXX=g++-10"
     - eval "export COV_COMPTYPE=gcc COV_PLATFORM=linux64"
+    - eval "export BUILD_TOOL_OPTIONS='-j 4'"
+    - eval "export GENERATOR='Unix Makefiles'"
   install:
-    - *install_coverity
```

Comment:
Why is this no longer needed?

Reply:
This is a typical example of why reviews are always a good idea... :relaxed: Thanks @eboasson!